scripts/convert.py

```diff
@@ -513,44 +513,22 @@ def get_replacement_dict(input_data: Dict[str, Any], mappings: bool = False) ->
 def get_replacement_mapping_value(k: str, v: str, el_text: str) -> str:
     reg_str: str = "^(OWASP SCP|OWASP ASVS|OWASP AppSensor|CAPEC|SAFECODE)\u2028" + k.replace("$", "\\$").strip() + "$"
     if re.match(reg_str, el_text.strip()):
-        pretext = el_text[: el_text.find("\u2028")]
-        v_new = v
-        if len(v) + len(pretext) > 60:
-            v_new = v.replace(", ", ",")
-        if len(v_new) >= 34:
-            v_split = v_new.find(",", 25 - len(pretext)) + 1
-            el_new = pretext + "    " + v_new[:v_split] + "\u2028" + v_new[v_split:].strip()
-            return el_new
+        if len(v) >= 38:
+            return el_text[: el_text.find("\u2028")] + ": " + v
         else:
             return el_text.replace(k, v)
     return ""
 
 
 def get_replacement_value_from_dict(el_text: str, replacement_values: List[Tuple[str, str]]) -> str:
     for k, v in replacement_values:
-        k2: str = k.replace("'", "’").strip()
-        v2: str = v.strip()
-        if el_text == k:
-            return v
-        elif el_text.strip() == k2:
-            return v2
-        elif el_text.lower() == k.lower():
-            return v
-        elif el_text.strip().lower() == k2.lower():
-            return v2
-        elif convert_vars.making_template:
```

Comment:
This code allowed templates to be created in the first place - it's probably no longer required and it exists in the git history if we ever need it again.

Reply:
Yes, I would not automaticly build templates. Keep in mind that we are talking about idml templates here, not yaml or docx